tests/test_functional.py

```diff
@@ -78,14 +78,22 @@ def get_tests():
 
 
 @pytest.mark.parametrize("test_file", TESTS, ids=TESTS_NAMES)
-def test_functional(test_file):
+def test_functional(test_file, benchmark):
+    benchmark.group = test_file._directory  # group benchmark results by module
     LintTest = (
         LintModuleOutputUpdate(test_file)
         if UPDATE
         else testutils.LintModuleTest(test_file)
     )
     LintTest.setUp()
-    LintTest._runTest()
+    benchmark_blacklist = (
```

Comment:
Maybe add a comment to explain why those are blacklisted, could be helpful to know if this is still relevant later :)